### ISSUE TITLE
VFS: Add iCloud Drive support

### ISSUE THREAD

**Issue by yanex** [marta-file-manager/marta-issues#315]

**Comment by haakonstorm** [comment]
Kind of works already! :-)
`~/Library/Mobile Documents/com~apple~CloudDocs` (at least on Catalina beta?)

**Comment by freetonik** [comment]
Yeah, you can navigate there (in Mojave, too, although the path is different: `~/Library/Mobile Documents/com~apple~CloudDocs/`). 

But Marta seem to not include this folder in its index, so you can't navigate there with `Cmd+P` lookup.

**Comment by IgorKrupenja** [comment]
I am on Sonoma 14.4.1 (23E224) and files in `/Users/igor/Library/Mobile Documents/com~apple~CloudDocs` are properly indexed. 🥳
